lm_eval/models/__init__.py

```diff
@@ -1,20 +1,11 @@
-from . import gpt2
-from . import gptj
-from . import gpt3
-from . import mgpt
-from . import t5
-from . import t0
 from . import dummy
+from . import openai_completion
+from . import huggingface
 
 MODEL_REGISTRY = {
-    "hf": gpt2.HFLM,
-    "gpt2": gpt2.GPT2LM,
-    "gptj": gptj.GPTJLM,
-    "gpt3": gpt3.GPT3LM,
-    "mgpt": mgpt.MGPTLM,
-    "t5": t5.T5LM,
-    "mt5": t5.T5LM,
-    "t0": t0.T0LM,
+    "hf-causal": huggingface.AutoCausalLM,
+    "hf-seq2seq": huggingface.AutoSeq2SeqLM,
+    "openai": openai_completion.CompletionLM,
```

Comment:
Why was `CompletionLM` chosen as the name for this?

Reply:
`Completion` is the name of the class we're using in the OpenAI python library. Their actual API endpoint is called "completion**s**" so maybe we could rename it `CompletionsLM` or `OpenAICompletionsLM` to be more explicit?